src/front/ticketer/lib/pages/page_organizer_register.dart

```diff
@@ -0,0 +1,253 @@
+import 'package:flutter/material.dart';
+
+class OrganizerRegisterPage extends StatefulWidget {
+  const OrganizerRegisterPage({Key? key}) : super(key: key);
+
+  @override
+  State<OrganizerRegisterPage> createState() => _OrganizerDataState();
+}
+
+class _OrganizerDataState extends State<OrganizerRegisterPage> {
+  final TextEditingController _companyName = TextEditingController();
+  final TextEditingController _addressCity = TextEditingController();
+  final TextEditingController _addressZip = TextEditingController();
+  final TextEditingController _addressStreet = TextEditingController();
+  final TextEditingController _taxId = TextEditingController();
+  final TextEditingController _displayName = TextEditingController();
+  final TextEditingController _phone = TextEditingController();
+
+  final _formKey = GlobalKey<FormState>();
+
+  Widget _companyNameEntryField() {
+    return TextFormField(
+      controller: _companyName,
+      decoration: const InputDecoration(
+          labelText: "Company name", hintText: 'Enter your company name'),
+      validator: (value) {
+        if (value == null || value.isEmpty) {
```

Comment:
I think there might be property `.isNull` if so, please replace ` == null ` with it

Reply:
I haven't found one